tests/test_pyenv_feature_exec.py

```diff
@@ -90,15 +90,6 @@ def test_exec_arg(self, setup, command, arg):
                 print(f'stderr: {stderr}')
             assert stdout == arg.replace('%World%', 'Earth')
 
-    def test_exec_set_var(self, setup):
```

Comment:
shims now create a temporary env to ensure path is not updated. So this unit tests must be removed (use case not supported anymore).